src/resources/views/product/index.blade.php

```diff
@@ -20,6 +21,8 @@
                 @endcan
             </div>
 
+            @include('vanilo::product._search')
```

Comment:
This file is missing from the PR

Reply:
Omg. You are so right. I have added it in.